sml_builder/templates/glossary.html

```diff
@@ -82,4 +82,12 @@ <h3><a name="{{ term.hash }}"></a>{{ term.term }}</h3>
 
 {%- block scripts -%}
     <script type=text/javascript src="{{ url_for('static',filename='js/glossary.js') }}"></script>
+    <!-- Google Tag Manager -->
```

Comment:
This doesn't have the cookies code? I think we would always track the glossary as things stand.

Reply:
Hi Joe took a look and noted one thing but suspects its to do with the outdated version and thinks his changes will correct this :)

Reply:
Hi Steve, so glossary does not seem to inherit the cookies code from core.html like the others even though it should. Just like the 'back to top' button does not work unless both scripts are added to the block script. This is tested and verified :)

Reply:
So I think we do need to fix this, it is not acceptable for us to ignore the cookie consent as it is part of GDPR etc. We need to find a way to add the cookie policy to the Glossary page, I don't think we can just say this page doesn't work like the others so we always track your usage.